frontends/election-manager/src/screens/write_ins_screen.tsx

```diff
@@ -1,147 +1,70 @@
-import React, { useContext, useEffect, useState } from 'react';
+import React, { useContext, useEffect, useMemo, useState } from 'react';
 import styled from 'styled-components';
 
 import { Button, Modal, Prose, Table, TD } from '@votingworks/ui';
 import {
-  Adjudication,
   CandidateContest,
   ContestId,
   getPartyAbbreviationByPartyId,
 } from '@votingworks/types';
 
+import { collections, groupBy } from '@votingworks/utils';
+import { Admin } from '@votingworks/api';
 import { NavigationScreen } from '../components/navigation_screen';
 import { WriteInsTranscriptionScreen } from './write_ins_transcription_screen';
 import { AppContext } from '../contexts/app_context';
 import { WriteInsAdjudicationScreen } from './write_ins_adjudication_screen';
+import { useWriteInsQuery } from '../hooks/use_write_ins_query';
```

Comment:
love that we don't have to boilerplate feed all these things through the context now.